packages/react-examples/src/react/TextField/TextField.Basic.Example.tsx

```diff
@@ -26,7 +26,12 @@ export const TextFieldBasicExample: React.FunctionComponent = () => {
         <TextField label="With an icon" iconProps={iconProps} />
         <TextField label="With placeholder" placeholder="Please enter text here" />
         <TextField label="Disabled with placeholder" disabled placeholder="I am disabled" />
-        <TextField label="Password with reveal button" type="password" canRevealPassword />
+        <TextField
+          label="Password with reveal button"
+          type="password"
+          canRevealPassword
+          revealPasswordAriaLabel="Reveal/hide password"
```

Comment:
I'd suggest having a simplified label here, along the lines of "Show password". The issue with something like "reveal/hide" is that when it's combined with a boolean attribute like `aria-pressed`, it's unclear what each state means. That is, "show password, on" is clear, but "reveal/hide password, on" is less so.